controllers/admin/AdminAjaxPrestashopCheckoutController.php

```diff
@@ -219,7 +219,7 @@ public function ajaxProcessPsxSendData()
 
         // Save form in database
         if (false === $this->savePsxForm($psxForm)) {
-            $this->ajaxDie(json_encode(false));
+            $this->ajaxDie(json_encode(['Cannot save in database.']));
```

Comment:
Shouldn't it be translatable ?

Reply:
Nope, this case should never happens and this whole shit will be gone in v2.0.0

Reply:
It save with Configuration::updateValue never return false, can return 0, 1 or true ^^